web/modules/custom/first_custom_module/first_custom_module.links.menu.yml

```diff
@@ -0,0 +1,19 @@
+first_custom_module.Page1:
+  title: 'Introduction'
+  description: 'This is the introduction page'
+  route_name: first_custom_module.Page1
```

Comment:
There are no quotes here in the RHS and in the next link you have added quotes.
Please try to follow a single approach.
Check in core files how it is done.